### ISSUE TITLE
Everpub Trademark 

### ISSUE THREAD

**Issue by Repositorian** [everpub/openscienceprize#117]
So in googling around for this openscienceprize everpub site, it became clear that there is another 'EverPub' in the digital publishing space (see search result screen cap below). This raises a pesky question of trademark / service marks. Realizing that legalities would be of little concern in this amazingly productive, creative visioning process unfolding, just throwing it out there for consideration in the future. Trademarks are about avoiding brand confusion, so could the multiple everpubs - if unleashed fall into that trap?

P.S. Librarians are not necessarily legal Debbie Downers, but IP stuff really does come into play a lot in our world, as long-term stewards of the scholarly record!
![everpub](https://cloud.githubusercontent.com/assets/17533421/13716884/b1f4ac64-e792-11e5-88a6-d0e3569717fb.jpg)

**Comment by ctb** [comment]
Right, I saw this, too ;). I think there's no problem in keeping the name for now (since everpub digital publishing solutions seems to have little web presence) and then any actual implementation can be named whatever else we want to name it.
